lib/src/document_init_parameters.dart

```diff
@@ -22,74 +22,73 @@ enum NativeImageDecoderSupport {
 
 class DocumentInitParameters {
   JsObject _jsInternal;
-  PDFDataRangeTransport _range;
+  PDFDataRangeTransport? _range;
 
-  DocumentInitParameters() {
+  DocumentInitParameters() :
     _jsInternal = JsObject.jsify({});
-  }
 
-  TypedData get data => _jsInternal['data'] as TypedData;
-  set data(TypedData data) {
+  TypedData? get data => _jsInternal['data'] as TypedData?;
+  set data(TypedData? data) {
     _jsInternal['data'] = data;
   }
 
-  String get docBaseUrl => _jsInternal['docBaseUrl'] as String;
-  set docBaseUrl(String docBaseUrl) {
+  String? get docBaseUrl => _jsInternal['docBaseUrl'] as String?;
+  set docBaseUrl(String? docBaseUrl) {
     _jsInternal['docBaseUrl'] = docBaseUrl;
   }
 
-  TypedData get initialData => _jsInternal['initialData'] as TypedData;
-  set initialData(TypedData initialData) {
+  TypedData? get initialData => _jsInternal['initialData'] as TypedData?;
+  set initialData(TypedData? initialData) {
     _jsInternal['initialData'] = initialData;
   }
 
-  Map<String, dynamic> get httpHeaders => _jsInternal['httpHeaders'] as Map<String, dynamic>;
-  set httpHeaders(Map<String, dynamic> httpHeaders) {
+  Map<String, dynamic>? get httpHeaders => _jsInternal['httpHeaders'] as Map<String, dynamic>?;
+  set httpHeaders(Map<String, dynamic>? httpHeaders) {
     _jsInternal['httpHeaders'] = httpHeaders;
   }
 
-  int get length => _jsInternal['length'] as int;
-  set length(int length) {
+  int? get length => _jsInternal['length'] as int?;
+  set length(int? length) {
     _jsInternal['length'] = length;
   }
 
-  NativeImageDecoderSupport get nativeImageDecoderSupport =>
+  NativeImageDecoderSupport? get nativeImageDecoderSupport =>
       _nativeImageDecoderSupportPdfjsToDart[_jsInternal['nativeImageDecoderSupport']];
-  set nativeImageDecoderSupport(NativeImageDecoderSupport nativeImageDecoderSupport) {
-    _jsInternal['nativeImageDecoderSupport'] = _nativeImageDecoderSupportDartToPdfjs[nativeImageDecoderSupport];
+  set nativeImageDecoderSupport(NativeImageDecoderSupport? nativeImageDecoderSupport) {
+    _jsInternal['nativeImageDecoderSupport'] = _nativeImageDecoderSupportDartToPdfjs[nativeImageDecoderSupport!];
   }
 
-  String get password => _jsInternal['password'] as String;
-  set password(String password) {
+  String? get password => _jsInternal['password'] as String?;
+  set password(String? password) {
     _jsInternal['password'] = password;
   }
 
-  PDFDataRangeTransport get range => _range;
-  set range(PDFDataRangeTransport range) {
-    _jsInternal['range'] = range._jsInternal;
+  PDFDataRangeTransport? get range => _range;
```

Comment:
`CustomDataRangeTransport` extends this class: https://github.com/Workiva/W_viewer/blob/4403afc058f41a04f07b4a58d5bd1afda3c41730/lib/src/modules/content_renderer/document_sources/helpers/custom_data_range_transport.dart#L14-L15